src/Logging/Logging.Console/src/ConsoleLoggerFormat.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    public enum ConsoleLoggerFormat
```

Comment:
Let's start this off right with some `///` doc comments describing the enum and both options.